src/components/Card/Card.jsx

```diff
@@ -18,6 +19,22 @@ function Card({
 }) {
   const [isDeleted, setIsDeleted] = useState();
 
+  const CERTIFICATED_SITE = [
+    "mozilla.org",
+    "react.dev",
+    "github.com",
+    "medium.com",
+    "velog.io",
+    "tistory.com",
+  ];
+
+  const isCertifiedSite = () => {
+    if (CERTIFICATED_SITE.some((site) => url.includes(site))) {
+      return true;
+    }
+    return false;
+  };
```

Comment:
```suggestion
  const isCertifiedSite = () => {
    return CERTIFICATED_SITE.some((site) => url.includes(site));
  };
```
이렇게 표현할 수 있을 것 같습니다!